pkgs/servers/xmpp/prosody/default.nix

```diff
@@ -43,6 +44,9 @@ stdenv.mkDerivation rec {
   # default setup.
   nixosModuleDeps = [
     "cloud_notify"
+    "cloud_notify_encrypted"
```

Comment:
This is only for the nixos module and will be reduced a lot with the 13.0.0 update